### ISSUE TITLE
'showCustomTime' - horizontally?

### ISSUE THREAD

**Issue by mschallar** [almende/vis#1427]
Wouldn't it be nice to have some kind of horizontal bar which can be added like 'showCustomTime'?

The workaround with 'add a second line-graph with no labels and two points' is not ideal.

**Comment by josdejong** [comment]
Yes that could be a new feature, maybe name it `showHorizontalLine`. I'm not sure if such a feature is interesting for many though. 

Would be nice if we could hook these sort of features into the Graph2d without having to adjust the code of Graph2d itself, to prevent Graph2d from growing to fat. We already have quite a modular approach, it may be not that difficult to support extensions like these.
